### PR TITLE
Completed javascript-forms.

### PR DESCRIPTION
**Pull request by stefancarrera**
Js Form Events
Js Form Values

### CODE REVIEW

File: javascript-forms/form-values.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var $contactForm = document.getElementById('contact-form');
+
+$contactForm.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var allValues = {};
+  allValues.name = document.forms[0].elements[0].value;
+  allValues.email = document.forms[0].elements[1].value;
+  allValues.message = document.forms[0].elements[2].value;
+  console.log(allValues);
```
Comment (scbowler) [inline javascript-forms/form-values.js:9]: You need to reset the form here

**Review by scbowler — CHANGES_REQUESTED** [review]
One change needed

File: javascript-forms/form-values.js (hunk 1 above)
Comment (stefancarrera) [reply javascript-forms/form-values.js:9]: added the form reset

**Review by scbowler — APPROVED** [review]
Great job! 👍

**Pull request merged**